IMPLEMENTATION.md

```diff
@@ -0,0 +1,119 @@
+# Tour of the Java classes
+
+The package `org.openmicroscopy.client.downloader` has the most classes
+including `Download` which carries the `main` method and various other
+methods relating to different setup and download options. Its
+`ALL_GROUPS_CONTEXT` constant is used by various classes to keep OMERO
+group context from limiting query results. The `main` method uses:
+
+1. The `org.openmicroscopy.client.downloader.options` package's
+   `OptionParser` which encodes the client application's command-line
+   options. That class uses `OptionSet` as a helper for options whose
+   value represents a subset of possible choices. An instance of
+   `OptionParser.Chosen` bears the options parsed from the command line.
+
+1. OMERO.blitz's `FindChildren` to determine which OMERO model objects
+   are targeted. The `RequestManager` is a helper class for submitting
+   requests to OMERO and ensuring that the response is of the expected
+   type. A `FileMapper` instance stores the outcome of queries to
+   understand how filesets on the server relates to their images and
+   pixels instances.
+
+1. `Download.downloadFiles` to implement the `-f` option's `binary` and
+   `companion`. This is where `FileManager` is used: for downloading
+   files from a raw file store provided by the server. (The class used
+   to have more methods.) It uses `LinkMakerPaths` for linking to the
```

Comment:
`(The class used to have more methods.)` Is this relevant to someone reading the docs today?

Reply:
I thought they might wonder if there's a reason why a single modest method gets a class all of its own.